builtin.c

```diff
@@ -474,6 +474,17 @@ XS(XS_builtin_indexed)
     XSRETURN(retcount);
 }
 
+XS(XS_builtin_load_module);
+XS(XS_builtin_load_module)
+{
+    dXSARGS;
+    if (items != 1)
+        croak_xs_usage(cv, "arg");
+    SV *st0 = ST(0);
```

Comment:
This line seems unnecessary, why not go straight for `SV *module_name = newSVsv(ST(0));`?

Reply:
Fair point. Rebased and squashed, thanks.